src/azure-cli/azure/cli/command_modules/resource/_help.py

```diff
@@ -2471,6 +2471,18 @@
     crafted: true
 """
 
+helps['resource patch'] = """
+type: command
+short-summary: Update a resource by PATCH request.
```

Comment:
We need to add more specific information for `resource update`, such as `Update a resource by PUT request`

Reply:
It is suggested to add a description of what conditions are suitable for `update` or `patch`, so as to reduce the learning and selection costs for customers